examples/seismic/viscoacoustic/operators.py

```diff
@@ -63,7 +62,85 @@ def blanch_symes(model, geometry, v, p, **kwargs):
     return [u_v, u_r, u_p]
 
 
-def ren(model, geometry, v, p, **kwargs):
+def sls_2nd_stencil(model, geometry, v, p, **kwargs):
```

Comment:
Ok, I see, but 1st and 2nd seems a bit vague for the code here. Are you able to be a bit more specific?

Reply:
We used `stencil` based on available examples. But we agree with you, `order` is better. We will change.